app/rax/src/client/preview/render.ts

```diff
@@ -29,6 +35,7 @@ export default function renderMain({
 
   showMain();
 
+  // @ts-ignore
```

Comment:
Maybe there is something weird here, more precisely on L23, as we are using the `storyFn` directly and not calling it, so `Element ` is a function but according to `createElement` types, there is no signature taking a function as input. However, as everything looks to work I didn't change anything to not break anything 🙈

Reply:
@merceyz suggested `@ts-expect-error` with a comment about why, which i think is a good idea

Reply:
Updated! ✅